uwblueprint/supportive-housing
# User and Residents Seeding Script

## Implementation description
* Working off of the changes in #74, I created a bash script to seed your own user into the DB. This was done because every time your database got reset, you had to delete your Firebase user and sign up again which was super tedious. With this script, you don't have to delete your Firebase user on a fresh DB. 
* I also added a script to seed Residents (since #50 was merged recently)



## Steps to test
### Setup
Add the following to your /seeding/.env file:
  ```
  # Residents Table
  RESIDENT_ROWS=5

  #User
  FIRST_NAME='<your first name>'
  LAST_NAME='<your last name>'
  AUTH_ID='<your auth id, can be found in the local DB>'
  ROLE='Admin'
  ```

### Testing
- Delete your user from the db (after you get your auth_id) and try to log in, verify you get a 500
- Run the script then try to log in again. Validate you can successfully log in
- Verify the `create-residents.sh` script runs correctly
- Verify the `create-log-records.sh` script is also working, since that was tweaked
